elements/rh-audio-player/rh-audio-player.ts

```diff
@@ -25,7 +27,9 @@ import { RhAudioPlayerScrollingTextOverflow } from './rh-audio-player-scrolling-
 import buttonStyles from './rh-audio-player-button.css';
 import rangeStyles from './rh-audio-player-range-styles.css';
 import styles from './rh-audio-player.css';
-import { RhTooltip } from '../rh-tooltip/rh-tooltip.js';
+
+import '../rh-surface/rh-surface.js';
+import '../rh-tooltip/rh-tooltip.js';
```

Comment:
We importing `rh-tooltip` in line 32 if we already did in line 1?

Reply:
b/c of type imports vs runtime